apis/python/tests/test_tiledbvcf.py

```diff
@@ -372,7 +372,9 @@ def test_read_null_attrs(tmp_path):
                                   np.array([89], dtype=np.int32), None,
                                   np.array([24], dtype=np.int32), None, None]),
             'fmt_DP': pd.Series([0, 0, 15, 64, 6, 2, 180,
-                                     88, 97, 24, 23, 21], dtype=np.int32)})
+                                     88, 97, 24, 23, 21], dtype=np.int32),
+            'fmt_MIN_DP': pd.Series([0, 0, 14, 30, 3, 1, 20,
+                                     None, 24, None, 23, 19], dtype=np.int32)})
```

Comment:
I think you need to change the dtype to `"Int32"` or `pd.Int32Dtype()`, per https://pandas.pydata.org/pandas-docs/stable/user_guide/integer_na.html

Reply:
Updated the `dtype` as discussed and noted issue in ch2928.